src/dht.cpp

```diff
@@ -2424,4 +2431,52 @@ Dht::onAnnounceDone(const Sp<Node>& node, net::RequestAnswer& answer, Sp<Search>
     sr->checkAnnounced(answer.vid);
 }
 
+
+void
+Dht::saveState(const std::string& path) const
+{
+    std::ofstream file(path);
+    msgpack::pack(file, exportNodes());
+    msgpack::pack(file, exportValues());
+}
+
+void
+Dht::loadState(const std::string& path)
+{
+    DHT_LOG.d("Importing state from %s", path.c_str());
```

Comment:
Note: these logs are not shown when using dhtnode.cpp, in fact loggers are enabled after `dht->run()` so loadState is already called.

Reply:
Indeed.. I plan to add the logger as part of the dht constructor or config (might be in another patch)

Reply:
yup not related to this PR (that's why I just add a Note). I've just noticed this weird thing.